api/newsletters/subscription_views.py

```diff
@@ -0,0 +1,41 @@
+from api.newsletters.newsletter_service import(address, get_subscription_list,add_to_subscription_list,check_subscription_list,remove_from_subscription_list)
+from api.newsletters.smtp import (send_newsletters,format_message)
+import json
+
+import logging
+from flask import (
+    Blueprint
+)
+from api.security.guards import (
+    authorization_guard,
+    permissions_guard,
+    admin_messages_permissions
+)
+
+
+bp_name = 'api-newsletter-subscription'
+bp_url_prefix = '/api/newsletter-subs'
```

Comment:
Minor thing here - for reuse of this /newsletter functionality in the future, you likely would create endpoints as /newsletter/subscriptions instead of /newsletter-subs - API standards usually don't favor adding a hyphen like that

Reply:
I see, very interesting I had no idea hyphens are a problem with APIs. We learn new things every day. Moreover, the naming convention was also off a bit. Will work on changing that @gregv